_haikus/hacktoberfest

```diff
@@ -0,0 +1,9 @@
+---
+layout: hailus
+title: Hacktoberfest
+Author: myself
+---
+
+*Its Hacktoberfest
+Time to make more pull reqests
+Lets all do our best*
```

Comment:
The asterisk `*` at the end of the line will be visible and won't achieve any formatting.